src/templates/main/index.html

```diff
@@ -139,9 +139,12 @@
                         equipment.<br><br>
                         We extend a cordial invitation to you to participate in Campus Placement Drive for the
                         season 2018-19 and hope that you will find the best talent you were looking for.
+                        {% if testimonial_list %} {% for Testimonial in testimonial_list %}
```

Comment:
What is this for?

Also wherever you have used `Testimonial` in html, refactor it to `testimonial`